### ISSUE TITLE
Uncaught TypeError: Cannot read property 'data' of undefined

### ISSUE THREAD

**Issue by jacekkopecky** [leny/atom-w3c-validation#1]
1. get a W3C validation report (happens to be valid)
2. close the pane by pressing the "x" icon on the right-hand side

**Atom Version**: 0.179.0
**System**: Mac OS X 10.9.5
**Thrown From**: [w3c-validation](https://github.com/leny/atom-w3c-validation) package, v0.1.3
### Stack Trace

Uncaught TypeError: Cannot read property 'data' of undefined

```
At /Users/jacek/.atom/packages/w3c-validation/node_modules/atom-message-panel/lib/MessagePanelView.js:206

TypeError: Cannot read property 'data' of undefined
  at [object Object].MessagePanelView.resizeStopped (/Users/jacek/.atom/packages/w3c-validation/node_modules/atom-message-panel/lib/MessagePanelView.js:206:6)
  at [object Object].MessagePanelView.close (/Users/jacek/.atom/packages/w3c-validation/node_modules/atom-message-panel/lib/MessagePanelView.js:108:12)
  at HTMLDivElement.<anonymous> (/Users/jacek/.atom/packages/w3c-validation/node_modules/atom-message-panel/node_modules/space-pen/lib/space-pen.js:214:36)
  at HTMLDivElement.jQuery.event.dispatch (/Users/jacek/.atom/packages/w3c-validation/node_modules/atom-message-panel/node_modules/space-pen/vendor/jquery.js:4676:9)
  at HTMLDivElement.elemData.handle (/Users/jacek/.atom/packages/w3c-validation/node_modules/atom-message-panel/node_modules/space-pen/vendor/jquery.js:4360:46)

```
### Commands

```
     -1:01.3 core:save (input.hidden-input)
     -0:07.2 command-palette:toggle (input.hidden-input)
     -0:05.9 core:confirm (input.hidden-input)
     -0:05.9 w3c-validation:validate (input.hidden-input)
     -0:03.5 vim-mode:reset-command-mode (input.hidden-input)
```

**Comment by leny** [comment]
Hi, thanks for the (_very complete_) report.

I'm quite busy right now, but I'll try to fix this current the week.

**Comment by jacekkopecky** [comment]
No worries. 8-)

**Comment by leny** [comment]
Hi, (sorry for the long delay).

I can't reproduce the error since I have updated the code to fix the Atom 1.0 API deprecation warning.

Seems to be fixed in `0.2.0`. Thanks for the report.
